raw-ipa-lib/src/user.rs

```diff
@@ -253,4 +265,42 @@ mod tests {
             User::point_from_matchkey(OTHER_PROVIDER, MATCHKEY.as_bytes())
         );
     }
+
+    #[test]
+    fn test_generate_event_report() {
```

Comment:
```suggestion
    fn generate_event_report() {
```

I'm not a fan of having all the functions in the "tests" module start with "test_" :)